client/app/components/AccountConnectionForm.jsx

```diff
@@ -3,14 +3,14 @@ import { h } from 'preact'
 import { translate } from '../plugins/preact-polyglot'
 
 import statefulForm from '../lib/statefulForm'
-import Field from './Field'
+import Field, { PasswordField } from './Field'
 
-const formConfig = ({ t, fields, accountName }) => {
+const formConfig = ({ t, fields, slug }) => {
   let values = {}
   Object.keys(fields).forEach(name => {
     if (fields[name].default) {
       values[name] = fields[name].default.replace(/<my_accounts>/gi, t('my_accounts title'))
-                                         .replace(/<account>/gi, accountName)
+                                         .replace(/<account>/gi, slug)
```

Comment:
The \<account\> is supposed to be the account __name__, __not the slug__. I replaced that in my last PR. Is there an issue somewhere else which expect the slug instead of the name? Why is it back?